package.json

```
@@ -60,8 +60,8 @@
     "test": "yarn lerna run --no-bail test",
     "test:c8-all": "rm -rf coverage/tmp && C8_OPTIONS=\"--clean=false --temp-directory=$PWD/coverage/tmp\" lerna run test:c8",
     "test:xs": "yarn workspaces run test:xs",
-    "build": "yarn workspaces run build",
-    "postinstall": "patch-package",
+    "build": "yarn workspaces run build && mkdir -p node_modules/.cache/agoric && date >node_modules/.cache/agoric/yarn-built",
+    "postinstall": "patch-package && mkdir -p node_modules/.cache/agoric && date >node_modules/.cache/agoric/yarn-installed",
```

this is quite a bit of boilerplate for package.json. please have a helper script.
```suggestion
    "build": "yarn workspaces run build && scripts/timestamp.sh yarn-built",
    "postinstall": "patch-package && scripts/timestamp.sh yarn-installed",
```

I wonder if there's a little npm package for reading and writing cached values or comparing timestamps with a cached value on disk.

---

I adopted this in `scripts/agd-builder.sh`.